e2e/tests/mobile/guest-shopper.spec.js

```diff
@@ -34,7 +34,10 @@ test("Guest shopper can checkout items as guest", async ({ page }) => {
 
   await clothingNav.click();
 
-  await page.getByRole("link", { name: "Tops" }).click();
+  const topsLink = page.getByRole("link", { name: "Tops" })
+  await topsLink.click();
+  // Wait for the nav menu to close first
+  await topsLink.waitFor({state: 'hidden'})
 
   await expect(page.getByRole("heading", { name: "Tops" })).toBeVisible();
```

Comment:
Also fixed this secondary bug.. previously we were actually asserting the Top link in the _nav menu_ (not the PLP page heading). Now we correct assert the correct thing, because we waited for the nav menu to close first.